src/app/components/home/add-marios/add-marios.component.html

```diff
@@ -0,0 +1,105 @@
+<div class="container">
+  <div class="back-button-container">
+    <a [routerLink]="['/home']">
+      <button class="back-button" type="button">
+        <img src="assets/icons/left-arrow.svg" alt="" />
+        BACK
+      </button>
+    </a>
+  </div>
+
+  <div class="form-container">
+    <form (ngSubmit)="onSubmit()" #f="ngForm">
+      <ng-select
+        [(ngModel)]="selected"
+        name="userSelect"
+        [multiple]="true"
+        class="user-select"
+        placeholder="Who you want to recognize?"
```

Comment:
FYI: placeholder text could be a variable
If in project there are a lots of placeholders, there could be file with texts in variables. But it's not this case